v6_carrier_py/master.py

```diff
@@ -177,6 +180,10 @@ def _combine_all_node_data(client, data, identifying_columns, *args, **kwargs) -
                                               subset=identifying_columns)
     n_dropped_rows = len_before_drop - len(combined_df)
     info(f'Dropped {n_dropped_rows} rows with duplicate identifiers')
+
+    if len(combined_df.index) < MIN_RECORDS:
+        raise Exception(f'Only {len(combined_df.index)} records available for analysis! Privacy is not ensured.')
```

Comment:
- Would be nice to also let the user know the MIN_RECORDS
- Better to raise something more specific than `Exception`, maybe `ValueError`?

Reply:
Done